test/models/promise.js

```diff
@@ -3,9 +3,17 @@ import { expect } from 'chai'
 import { promiseGallerySort } from '../../models/promise'
```

Comment:
if we move the sort function to it's own file, this would become like `import promiseGallerySort from '../../lib/sort'` (use `export default` and remove the `{}` from the import), and this test file should move to match (`test/lib/sort.js`)